### PR TITLE
takebuf_string(...) => String(take!(...))

### PR DESCRIPTION
**Pull request by dmbates**
to squelch warnings in v0.6.0

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,2 +1,2 @@
-julia 0.4
-Compat
+julia 0.5
+Compat 0.18
```
Comment (tkelman) [inline REQUIRE]: 0.9.5 for take!, and it doesn't need the at-compat macro since it's not a syntax tranformation

Comment (dmbates) [reply REQUIRE]: Thanks Tony.  I am constantly amazed at the number of tasks, like checking this pull request, that you can accomplish simultaneously

**Review by tkelman — APPROVED** [review]
lgtm, remind me to merge in a bit if no other feedback

**Comment by dmbates** [comment]
@tkelman This is your reminder to merge.  If I create a new release (I have accepted your invitation to join JuliaGraphics) do I need to put an upper bound on the version of julia in the REQUIRE files for earlier releases?  I think they will still work, it is just that they produce deprecation warnings.

**Comment by tlnagy** [comment]
I'm in the process of tagging a new release of `Showoff` for `Gadfly`. LGTM

**Pull request merged**